iosApp/iosApp/Pages/TripDetails/VehicleCardView.swift

```diff
@@ -78,10 +78,18 @@ struct VehicleOnTripView: View {
                     .font(Typography.headlineBold)
                     .foregroundColor(textColor)
             }
+            .accessibilityElement()
+            .accessibilityAddTraits(.isHeader)
+            .accessibilityHeading(.h2)
+            .accessibilityLabel(
+                "\(route.type.typeText(isOnly: true)) \(vehicleStatusDescription(vehicle.currentStatus)) \(stop.name)"
+            )
```

Comment:
suggestion: if `accessibilityElement` is used with [.combine](https://developer.apple.com/documentation/swiftui/accessibilitychildbehavior/combine), then the vehicle type accessibility label can be moved to the vehicle icon and the status + stop name will be read as specified by those views. As-is, the vehicle icon is read out loud as "small subway image"

Reply:
I see how your point above about creating a new combined label applies in this case, but something funky seems to be going on - using voice over I hear "train underscore conditional content..."

Reply:
Hmm, thanks for that. I'll look into it.